lua/entities/gmod_wire_pod.lua

```diff
@@ -13,42 +12,47 @@ if CLIENT then
 	local toolgunHUDFunc = nil
 	local function blank() end
 
-	usermessage.Hook( "wire pod hud", function( um )
-		hideHUD = um:ReadShort()
+	net.Receive("wire_pod_hud", function()
+		hideHUD = net.ReadUInt(2)
 		if hideHUD > 0 and not HUDHidden then
+			local ply = LocalPlayer()
 			HUDHidden = true
+
 			if firstTime then
-				LocalPlayer():ChatPrint( "The owner of this vehicle has hidden your hud using a pod controller. If it gets stuck this way, use the console command 'wire_pod_hud_show' to forcibly enable it again." )
+				ply:ChatPrint("The owner of this vehicle has hidden your hud using a pod controller. If it gets stuck this way, use the console command 'wire_pod_hud_show' to forcibly enable it again.")
 				firstTime = false
 			end
-			--Hide toolgun HUD
-			local toolgun = LocalPlayer():GetWeapon("gmod_tool")
+
+			-- Hide toolgun HUD
+			local toolgun = ply:GetWeapon("gmod_tool")
 			if IsValid(toolgun) then
 				toolgunHUDFunc = toolgun.DrawHUD
 				toolgun.DrawHUD = blank
 			end
-			--Hide all HUDPaints except for EGP HUD
+
+			-- Hide all HUDPaints except for EGP HUD
 			local hooks = hook.GetTable()["HUDPaint"]
 			savedHooks = table.Copy(hooks)
 			for k in pairs(hooks) do
 				if hideHUD > 2 or k ~= "EGP_HUDPaint" then
-					hook.Add( "HUDPaint", k, blank )
+					hook.Add("HUDPaint", k, blank)
 				end
 			end
-			--Hide other HUD elements
-			hook.Add( "DrawDeathNotice", "Wire pod DrawDeathNotice", function() return false end)
-			hook.Add( "HUDDrawTargetID", "Wire pod HUDDrawTargetID", function() return false end)
-			hook.Add( "HUDShouldDraw", "Wire pod HUDShouldDraw", function( name )
+
+			-- Hide other HUD elements
+			hook.Add("DrawDeathNotice", "wire_pod_drawdeathnotice", function() return false end)
+			hook.Add("HUDDrawTargetID", "wire_pod_huddrawtargetid", function() return false end)
+			hook.Add("HUDShouldDraw", "wire_pod_hudshoulddraw", function(name)
 				if hideHUD > 0 then
-					if LocalPlayer():InVehicle() then
+					if ply:InVehicle() then
```

Comment:
Need `local ply = LocalPlayer()` in this hook function. The stored one may go invalid